src/test/java/com/puppycrawl/tools/checkstyle/api/AbstractFileSetCheckTest.java

```diff
@@ -263,6 +264,20 @@ public void testMultiFileFireErrors() throws Exception {
                 .isEqualTo(2);
     }
 
+    /**
+     * S2384 - Mutable members should not be stored or returned directly.
```

Comment:
Please expand comment similar to https://github.com/checkstyle/checkstyle/pull/12023#discussion_r938703126

Reply:
Done.